cmd/vela-server/main.go

```diff
@@ -151,6 +151,17 @@ func main() {
 			Usage:   "sets the level of compression for logs stored in the database",
 			Value:   constants.CompressionThree,
 		},
+		&cli.IntFlag{
```

Comment:
This looks like duplicate code

Reply:
Thanks for finding that!

I believe this has been fixed 🎉